tqp/topic_queue_poller.py

```diff
@@ -17,17 +17,23 @@ def _jsonify_dictionary(dictionary):
 # -----------------------------------------------------------------------------
 
 
-def create_queue(queue_name, **kwargs):
+def create_queue(queue_name, tags=None, **kwargs):
+    tags = tags or {}
     sqs = boto3.resource("sqs")
 
-    def _create_queue(name, attributes):
+    def _create_queue(name, attributes, extra_tags=None):
+        extra_tags = extra_tags or {}
+
         return sqs.create_queue(
-            QueueName=name, Attributes=_jsonify_dictionary(attributes),
+            QueueName=name,
+            Attributes=_jsonify_dictionary(attributes),
+            tags={"TQP": "true", **extra_tags, **tags},
```

Comment:
```suggestion
            tags={"TQP": "true", **tags, **extra_tags},
```

presumably these should take precedence?